lec01-introduction.Rmd

```diff
@@ -2,3 +2,171 @@
+
+So if I create a script with some of the commands you saw earlier, I can save them all as a text file and then run the entire script by clicking the run button, which executes commands in the console. We will use this text editor to explore how to make plots in R.
+
+## Plots
+In R, it is straightforward to make explore your data by creating nice looking plots. E.g. to make a scatter plot, we first tell R that we want to make a plot of the `iris` data frame, and we want the aesthetics of this plot to be that `sepal_length` is on the x axis and `sepal_width` is on the y-axis. Finally, we specify that we want this to be a plot of points, i.e. a scatter plot.
+
+```{r}
+ggplot(iris, aes(x=sepal_length, y=sepal_width)) +
+    geom_point()
+```
+
+This plots shows up in the plot panel in RStudio, which is convenient instead of having it show up as a separate window. You can already see that there seems to be at least some structure to the data. There is one cluster of points up the left corner and at least one in the lower right corner. Hmmmm, I wonder if these correspond to the different species of flowers... Let's find out, by coloring the points according to which species they are.
+
+```{r}
+ggplot(iris, aes(x=sepal_length, y=sepal_width, color=species)) +
+    geom_point()
+```
+
+Likewise, it would be really easy to make a histogram of the petal_length and color it according to the species.
+
+```{r}
+ggplot(iris, aes(x=petal_length, fill=species)) +
+    geom_histogram()
+```
+
+And we could get fancier by simply instructing R to split this plot in vertically stacked facets, or subplots, with one species in each facet.
+
+```{r}
+ggplot(iris, aes(x=petal_length, fill=species)) +
+    geom_histogram() +
+    facet_wrap(~ species, dir='v')
+```
+
+One last example If you don't have a clear idea of what you are looking for in a data set, maybe you are exploring publicly available data that you have not generated yourself, then it can be useful to visualize the relationships between all the variables to get an overview of the data.
+
+```{r}
+ggpairs(iris, aes(colour = species))
+```
+
+This plot is a bit overwhelming, and the purpose here is not to produce a publication ready figure. You can see that many of the labels are overlapping and not everything fits in the plotting window. We can tweak these properties, which we will get into later in the course. But even in its slightly rough form, this is a powerful data set overview for your own analyses.
+
+Hopefully, these examples have giving you a idea of how simple and powerful it is to use R for you exploratory data analyses. In a few lines of codes we have made some really nice plots, some of which are not even imaginable to create in Excel.
```

Comment:
s/you exploratory/your exploratory/

Reply:
I believe missing the "r" in "your" is my most common typo...